### PR TITLE
Bug 1168951 - Pin Page

### PR DESCRIPTION
**Pull request by benfrancis**

### CODE REVIEW

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=ce589027022533eee723a877a872ffc09447bffb

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=6a02d41b6da8d4ccae81a4fad5aea60bb3ea0ab2

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=71f7582f9254493aae7eeb50f0c966b974ee63b5

File: apps/system/test/unit/app_chrome_test.js [hunk 1]
```diff
@@ -978,10 +978,10 @@ suite('system/AppChrome', function() {
       getIconPromise.mRejectToError();
 
       assert.ok(combinedChrome.siteIcon
-                  .style.backgroundImage.includes(fakeIconURI));
+                  .style.backgroundImage.indexOf(fakeIconURI) != -1);
     });
 
-    test('has no effect for private browsers', function() {
+    test('has no e1168951ffect for private browsers', function() {
```
Comment (albertopq) [inline apps/system/test/unit/app_chrome_test.js]: I think this change wasn't supposed to be here :)

File: apps/system/js/app_chrome.js [hunk 2]
```diff
@@ -357,9 +360,26 @@
...
+        this.pin();
+        break;
     }
   };
 
+  AppChrome.prototype.pin = function ac_pin() {
+    this.hidePinDialogCard();
+    this.collapse();
+    this.pinned = true;
+    this.app.element.classList.remove('collapsible');
```
Comment (albertopq) [inline apps/system/js/app_chrome.js]: Why the chrome is not collapsible anymore after pinning?

File: apps/system/test/unit/app_chrome_test.js (hunk 1 above)
Comment (benfrancis) [inline apps/system/test/unit/app_chrome_test.js]: Oops, silly text editor! :)

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=91fa3a24b40e9db1f8dea6a219d83eedac51d515

**Pull request merged**